### ISSUE TITLE
`okteto down` doesn't scale up again original deployments when using `services` key in the manifest

### ISSUE THREAD

**Issue by ifbyol** [okteto/okteto#3363]
**Describe the bug**
When using `services` key in the Okteto manifest under `dev` section (https://www.okteto.com/docs/reference/manifest/#services-object-optional), if you execute `okteto down` to stop dev mode, original deployments for all the services specified there are not scaled up to the original number of replicas

**To Reproduce**

1. Clone repository https://github.com/ifbyol/go-getting-started
2. Go to branch `ifbyol/services-manifest`
3. Execute `okteto deploy`
4. When finished, execute `okteto up`. If you check the deployments in your ns at that moment, you should see something like this (the original deployment for the service specified under `services` key is scale to 0 and the dev one has 3 replicas)

``` bash
╰─ k get deployments -n nacho
NAME                 READY   UP-TO-DATE   AVAILABLE   AGE
dev-okteto           1/1     1            1           38s
hello-world          0/0     0            0           76s
hello-world-okteto   3/3     3            3           38s
```

5. If you execute now `okteto down`, you will see something like this (dev deployments are gone and the original one wasn't scale to the right number of replicas):

```bash
╰─ k get deployments -n nacho
NAME          READY   UP-TO-DATE   AVAILABLE   AGE
hello-world   0/0     0            0           82s
```

**Expected behavior**
After executing `okteto down`, the original deployment should end up in the same state it was before executing the `okteto up`

**Desktop (please complete the following information):**

- Version: It happens at least since version 2.1

**Additional context**

It happens also in one of our examples: https://github.com/okteto/samples/tree/main/php

**Comment by AnshulAngaria** [comment]
@ifbyol I have made the fix in the PR #3556 .

**Comment by pchico83** [comment]
Closed by https://github.com/okteto/okteto/pull/3556
